app/src/main/java/com/cornellappdev/volume/views/WebviewTop.kt

```diff
@@ -34,7 +34,7 @@ class WebviewTop @JvmOverloads constructor(
     }
 
     fun setName(article: Article) {
-        name.text = if (article.title!!.length < 40) article.title else article.title.substring(0, 41) + "..."
```

Comment:
I think this was causing the crashes Lesley was experiencing (as it was crashing for me clicking on articles w/ exactly 41 characters, will be more explicit on some crash screen however